docs/volunteering/mentor-guidance.md

```diff
@@ -0,0 +1,223 @@
+fine to leave explaining those until they become important.
+
+A superb example of this in action is this video [*explaining quantum
+computing at five levels of depth*](https://www.youtube.com/watch?v=OWJCfOvochA).
+
+### Explaining the “how”
+
+Of course after a while many students will want to understand the “how”
+of something as well as the “what”. The point where they start wanting
+to know this varies from person to person as does the impact it has on
+their learning.
+
+As a result, while a hand-waved explanation (ignoring the man behind the
+curtain) will suffice when initially explaining a topic, you should be
+willing to explain in more detail if the student asks. Bear in mind that
+you still need to find the right level on this axis too -- if you switch
+to a very detailed explanation then it can still bewilder the learner.
+
+For example, in the case of our variable assignment, you might be asked
+how the computer understands the idea of 42, or how it’s stored. A first
+response might be to talk about the differences between numbers (used
+for addition, etc.) and strings (used for text). Even if you do end up
+talking about the different in-memory representations (perhaps the
+learner has already heard about ASCII), you’d almost certainly want to
+avoid talking about the internals of CPython objects.
+
+This line of questioning is sometimes related to the student trying to
+attach or relate the new topic so something they already know, but not
+being sure what the connections are. It’s reasonable to ask the student
+if this is the case, either by asking why they’re asking for more
+details (“what makes you ask that?”) or if they’ve seen something
+similar before.
+
+## Learn by doing
+
+As much as possible, encourage the learner to experiment with what
+they’ve just learned. Learning is reinforced by putting it into practise
+and is often most enjoyable when discovering new things out for
+yourself.
+
+For our learning Python example, this means getting the learner into a
+Python shell or similar environment where they can easily try new
+things. It doesn’t matter that they’re not writing a full program yet,
+the feedback loop from trying things and experiencing that there’s no
+downside to things not working as hoped is what you’re after.
+
+## Keep it interactive
+
+Initially when explaining something you’ll need to mostly be telling the
+learner what things are and what they do. At this stage, it’s important
+to gauge their reaction to ensure you’re going at a pace that works for
+them. If there’s something you’re not sure that they’re understanding,
+do ask them!
+
+Once the learner has formed enough of a basis of understanding of a
+topic you can start asking them to explain how they think things might
+work if there were slight variations. In our Python example, you might
+show a conditional block:
+
+``` python
+foo = 42
+if foo > 25:
+    print('greater')
+print('nope')
```

Comment:
Are you missing an `else` here?

Reply:
probably, that's entirely not the point though.